var/spack/repos/builtin/packages/vc/package.py

```diff
@@ -20,3 +21,10 @@ class Vc(CMakePackage):
             description='The build type to build',
             values=('Debug', 'Release', 'RelWithDebug',
                     'RelWithDebInfo', 'MinSizeRel'))
+    variant('test', default=False, description='Build tests')
+    
+    def cmake_args(self):
+        if "+test" in self.spec:
```

Comment:
```suggestion
        if self.run_tests:
```
Then you can get rid of the variant. That way, regardless if you run `spack install --test=root vc` or `spack install vc`, it will only install a single package.